packages/meditrak-server/src/routes/importSurveys/importSurveys.js

```diff
@@ -389,7 +389,26 @@ async function processOptionSetName(models, name) {
   return null;
 }
 
-function processOptions(optionValuesString, optionLabelsString, optionColorsString) {
+const processOptions = (optionValuesString, optionLabelsString, optionColorsString, type) => {
+  switch (type) {
+    case 'Binary':
+      return processBinaryOptions(optionLabelsString, optionColorsString);
+    default:
+      return processDefaultOptions(optionValuesString, optionLabelsString, optionColorsString);
+  }
+};
+
+const processBinaryOptions = (optionLabelsString, optionColorsString) => {
+  const optionLabels = splitOnNewLinesOrCommas(optionLabelsString);
+  const optionColors = splitOnNewLinesOrCommas(optionColorsString);
+  return optionLabels.map((label, i) => {
+    const color = optionColors[i];
+
+    return { label, color };
```

Comment:
What about adding `value` here, and making the code in meditrak-app simpler? That way instead of mapping through ['Yes', 'No'], you could just pass the options array created here directly.

Reply:
I was thinking of doing the same, but then I chose the current way because I kinda didn't want to store the `value` Yes/No in the `options` column (which can be changed through the admin panel). Is that a good enough reason or maybe I was just overthinking? But I agree your suggestion will make things much easier!

Reply:
Never mind I was definitely overthinking, will follow your suggest!

Reply:
That's a good thought...probably best solution would be validation on editing options through the admin panel. Let's just leave it for now though, because that is sure to create conflicts with Ethan's work to add the api level permissions